packages/valory/skills/abstract_round_abci/test_tools/abci_app.py

```diff
@@ -96,54 +96,26 @@ def process_payload(self, payload: BaseTxPayload) -> None:
         """Process payloads of type 'payload_background_c'."""
 
 
-class ConcreteTerminationRoundA(AbstractRound):
+class ConcreteTerminationRoundA(ConcreteRoundA):
     """Dummy instantiation of the AbstractRound class."""
 
     round_id = "concrete_termination_a"
     allowed_tx_type = "payload_termination_a"
 
-    def end_block(self) -> Tuple[MagicMock, MagicMock]:
-        """End block."""
-        return MagicMock(), MagicMock()
 
-    def check_payload(self, payload: BaseTxPayload) -> None:
-        """Check payloads of type 'payload_termination_a'."""
-
-    def process_payload(self, payload: BaseTxPayload) -> None:
-        """Process payloads of type 'payload_termination_a'."""
```

Comment:
asked the author about the purpose of returning `MagicMock(), MagicMock()`, was informed it was because it was a copy paste of `ConcreteRoundA`. Hence didn't write a test for coverage and refactored instead.